OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -362,7 +362,7 @@ public override bool HandleMouseInput(MouseInput mi)
 				else if (mi.Event == MouseInputEvent.Move && (isStandardScrolling ||
 					(standardScrollStart.HasValue && ((standardScrollStart.Value - mi.Location).Length > Game.Settings.Game.MouseScrollDeadzone))))
 				{
-					isStandardScrolling = true;
+					if (scrollType == MouseScrollType.Standard) isStandardScrolling = true;
```

Comment:
You can change this to
```c#
isStandardScrolling = scrollType == MouseScrollType.Standard;
```

Reply:
You are a wise man, my friend!

I updated it.